packages/core/src/command/hl7v2-subscriptions/hl7v2-roster-generator.ts

```diff
@@ -127,6 +128,13 @@ export class Hl7v2RosterGenerator {
     });
 
     log(`Saved in S3: ${this.bucketName}/${fileName}`);
+
+    const uploadTimer = initTimer();
+
+    log(`Starting SFTP upload for config: ${config.name}`);
+    await uploadThroughSftp(config, rosterCsv);
+    log(`SFTP upload completed in ${uploadTimer.getElapsedTime()}ms`);
```

Comment:
small comments - group the uploadTimer with the rest of the upload code

& this is my personal pref but...
keep the log at the top of the block (start the timer after the log), it acts like a comment / header for the section of code.